document_page/views/document_page_category.xml

```diff
@@ -109,132 +106,86 @@
-                                            href="#"
-                                            aria-label="Dropdown menu"
-                                            title="Dropdown menu"
-                                        >
-                                            <span class="fa fa-ellipsis-v" />
-                                        </a>
-                                        <div class="dropdown-menu" role="menu">
-                                            <ul
-                                                class="oe_kanban_colorpicker"
-                                                data-field="color"
-                                            />
-                                        </div>
-                                    </div>
-                                    <div class="o_kanban_record_body">
-                                        <div class="o_kanban_tags_section">
-                                        </div>
-                                    </div>
-                                    <div
-                                        class="o_kanban_record_bottom"
-                                        t-if="record.type.raw_value == 'content'"
-                                    >
-                                        <div class="oe_kanban_bottom_left">
-                                            <field name="write_date" widget="date" />
-                                        </div>
-                                        <div class="oe_kanban_bottom_right">
-                                            <img
-                                                t-att-src="kanban_image('res.users', 'image_small', record.content_uid.raw_value)"
-                                                t-att-title="record.content_uid.value"
-                                                t-att-alt="record.content_uid.value"
-                                                width="24"
-                                                height="24"
-                                                class="oe_kanban_avatar"
-                                            />
-                                        </div>
-                                    </div>
-                                </div>
-                            </div>
+                                <i
+                                    t-if="!record.image.raw_value"
+                                    class="o_field_image fa fa-folder-open"
+                                    style="font-size: 64px; color: lightslategray"
+                                />
+                            </t>
+                            <t t-if="record.type.raw_value == 'content'">
+                                <i
+                                    class="o_field_image fa fa-file"
+                                    style="font-size: 64px; color: lightgray"
+                                />
+                            </t>
```

Comment:
We shouldn't discriminate against record.type.raw_value == 'content' like that. Let his image be displayed.
```suggestion
<t t-if="record.type.raw_value == 'category'">
                                <field
                                        name="image"
                                        t-if="record.image.raw_value"
                                        widget="image"
                                        t-att-alt="record.display_name.raw_value"
                                    />
                                <i
                                        t-if="!record.image.raw_value"
                                        class="o_field_image fa fa-folder-open"
                                        style="font-size: 64px; color: lightslategray"
                                    />
                            </t>
                            <t t-if="record.type.raw_value == 'content'">
                                <field
                                        name="image"
                                        t-if="record.image.raw_value"
                                        widget="image"
                                        t-att-alt="record.display_name.raw_value"
                                    />
                                <i
                                    t-if="!record.image.raw_value"
                                    class="o_field_image fa fa-file"
                                    style="font-size: 64px; color: lightgray"
                                />
                            </t>
```